src/lib/components/Button/Button.scss

```diff
@@ -57,104 +44,84 @@ $filters: (
 
 .usa-button {
   width: auto;
-  color: #fff;
-  background: map.get($colors, "primary");
   .usa-icon {
     filter: map.get($filters, "white");
   }
-  &:hover {
-    background: map.get($colors, "primary-dark");
-  }
-  &:active {
-    background: map.get($colors, "primary-darker");
-  }
   &:disabled,
   &:disabled:hover,
   &:disabled:active {
-    background: map.get($colors, "disabled-dark");
+    // Default is $disabled, but we want a shade darker.
+    background: $disabled-dark;
   }
 }
 
 .usa-button.usa-button--secondary {
-  color: map.get($colors, "primary");
-  background: map.get($colors, "secondary");
+  color: $primary;
   .usa-icon {
     filter: map.get($filters, "primary");
   }
-  &:hover {
-    background: map.get($colors, "secondary-dark");
-  }
-  &:active {
-    background: map.get($colors, "secondary-darker");
-  }
-  &:disabled,
-  &:disabled:hover,
-  &:disabled:active {
-    background: map.get($colors, "disabled-dark");
+  &:disabled {
+    color: $white;
+    .usa-icon {
+      filter: map.get($filters, "white");
+    }
   }
 }
 
 .usa-button.usa-button--base {
-  background: map.get($colors, "base-dark");
-  &:hover {
-    background: map.get($colors, "base-dark");
-  }
```

Comment:
Hover state on the base button should show no change. Adding a rule here would be redundant.